libgadget/blackhole.c

```diff
@@ -460,10 +461,7 @@ blackhole_accretion_ngbiter(TreeWalkQueryBHAccretion * I,
             double mass_j;
             if(HAS(All.BlackHoleFeedbackMethod, BH_FEEDBACK_OPTTHIN)) {
                 double redshift = 1./All.Time - 1;
-                double InternalEnergy = DMAX(All.MinEgySpec, SPHP(other).Entropy / GAMMA_MINUS1 * pow(SPHP(other).EOMDensity * All.cf.a3inv, GAMMA_MINUS1));
-                double physdens = SPHP(other).Density * All.cf.a3inv;
-                struct UVBG uvbg = get_local_UVBG(redshift, P[other].Pos);
-                double nh0 = GetNeutralFraction(InternalEnergy, physdens, &uvbg, SPHP(other).Ne);
+                double nh0 = get_neutral_fraction_sfr(other, redshift);
```

Comment:
This is fine. I don't think we use this in production. It was a very crude extreme case scenario that we do not think is of reality.